firedrake/interpolation.py

```diff
@@ -260,7 +261,8 @@ def compile_ufl_kernel(expression, to_pts, to_element, fs):
     args.insert(0, ast.Decl("double", ast.Symbol('A', rank=(len(to_pts),) + fs.shape)))
     kernel_code = builder.construct_kernel("expression_kernel", args, body)
 
-    return op2.Kernel(kernel_code, kernel_code.name), oriented, coefficients
+    return op2.Kernel(kernel_code, kernel_code.name, opts=parameters["coffee"]), \
```

Comment:
This is one step to get the interpolation kernels COFFEE-optimized, but few others are also required:
 * Generate COFFEE AST with the appropriate `#pragma coffee` annotations, so that COFFEE will realise that it has work to do.
 * Make sure that the TSFC codegen options that COFFEE relies on are turned on (currently the expansion of geometric `IndexSum`s).
 * Make sure there are meaningful optimisations that COFFEE can do on an interpolation kernel. (Since it was designed with forms in mind.)